cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -67,7 +69,8 @@ public AbstractSchedulerService(Supplier<org.quartz.Scheduler> schedulerSupplier
    */
   protected final void startScheduler() {
     try {
-      delegate.start();
+      timeScheduler.start();
+      streamSizeScheduler.start();
```

Comment:
would you want to log what scheduler(s) were started? And same below for stopped.